geomesa-process/geomesa-process-vector/src/main/scala/org/locationtech/geomesa/process/tube/TubeBuilder.scala

```diff
@@ -156,6 +156,56 @@ class LineGapFill(tubeFeatures: SimpleFeatureCollection,
 
   def nextId: String = id.getAndIncrement.toString
 
+  /**
+    * This function checks if a segment crosses the IDL.
+    * @param point1 The first point in the segment
+    * @param point2 The second point in the segment
+    * @return boolean representing validity of a segment based on segment length
+    */
+  def crossesIDL(point1:Coordinate,point2:Coordinate): Boolean = {
+    return (Math.abs(point1.x-point2.x) > 350)
+  }
+
+  /**
+    * Calculate the latitude at which the segment intercepts the IDL.
+    * @param point1 The first point in the segment
+    * @param point2 The second point in the segment
+    * @return a dobule representing the intercept latitude
+    */
+  def calcIDLIntercept(point1: Coordinate, point2: Coordinate): Double = {
+    if(point2.x > point1.x){
+      return point2.y - (((point2.y-point1.y)/(point2.x-(-180+point1.x)))*point2.x)
```

Comment:
for scala, we prefer to omit the 'return' keyword. in this case, you can just remove it and put the second return in an else branch

Reply:
All instances of return are now removed, else is used where appropriate. Thank you for the guidance on that.